src/pages/staticPages/webAPI/WebAPIPage.tsx

```diff
@@ -622,7 +699,9 @@ Error:  No case lists available for cancer_study_id:  gbs.
                 </p>
                 <p>And a link to TP53 mutations across all cancer studies:</p>
                 <p><a href="ln?q=TP53:MUT">ln?q=TP53:MUT</a></p>
-        </PageLayout>;
+                {this.renderDataAccessTokensDiv()}
```

Comment:
I think this section should come after the Introduction, so that users are aware of the need to use tokens before they try to jump in and access the service.